terraform/system-apps/log-integration/log-integration/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
Please move the component to new directory terraform/integration-demos/ (please update manage-env.sh as well)